.env.development

```diff
@@ -4,12 +4,12 @@ REACT_APP_ALGOLIA_APPPLICATION_ID=AJCS0S8FVN
 REACT_APP_ALGOLIA_INDEX=SEARCH_EM
 
 # FIREBASE CONFIG
-REACT_APP_FIREBASE_API_KEY=AIzaSyBpC2KffoQNufewJ4uS-WS72sqeXVJ5T-U
-REACT_APP_FIREBASE_AUTH_DOMAIN=em-search-test.firebaseapp.com
-REACT_APP_FIREBASE_DATABASE_URL=https://em-search-test-default-rtdb.firebaseio.com/
-REACT_APP_FIREBASE_PROJECT_ID=em-search-test
-REACT_APP_FIREBASE_STORAGE_BUCKET=gs://em-search-test.appspot.com/
-REACT_APP_FIREBASE_MESSAGE_SENDER_ID=9194796048
+REACT_APP_FIREBASE_API_KEY=AIzaSyA9J3iBGSFK3qMdHA0G-G3kY_FFBZ5Lujg
+REACT_APP_FIREBASE_AUTH_DOMAIN=em-search-test-3bfe3.firebaseapp.com
+REACT_APP_FIREBASE_DATABASE_URL=https://em-search-test-3bfe3-default-rtdb.firebaseio.com/
+REACT_APP_FIREBASE_PROJECT_ID=em-search-test-3bfe3
+REACT_APP_FIREBASE_STORAGE_BUCKET=em-search-test-3bfe3.appspot.com
+REACT_APP_FIREBASE_MESSAGE_SENDER_ID=976083438599
 REACT_APP_FIREBASE_REDIRECT_URL=http://www.localhost:3000
```

Comment:
Please revert and use `.env.development.local` for your personal Firebase config.